k9mail/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -194,6 +194,7 @@
     private Action action;
 
     private boolean requestReadReceipt = false;
+    private boolean isHighPriority = false;
```

Comment:
I feel like this belongs in `RecipientPresenter` (which should really be `MetadataPresenter`), but I suppose having it next to requestReadReceipt makes sense as well. feel free to leave as is.